gems/git-fastclone/OSVDB-1.yml

```diff
@@ -0,0 +1,11 @@
+---
+gem: git-fastclone
+osvdb: 
+url: https://github.com/square/git-fastclone/pull/5
+title: |
+  git-fastclone Improper use of cocaine allows for malicious common injection
+date: 2015-12-15
+description: |
+  Fastclone is using cocaine improperly, allowing malicious command injection.
```

Comment:
I looked at the GitHub Fix Commit and came up with this description https://srcclr.com/security/command-injection/ruby/s-1922